docs/dasharo-tools-suite/documentation/supported-hardware.md

```diff
@@ -0,0 +1,17 @@
+# Supported hardware
+
+Dasharo Tools Suite was prepared to run on x86 platforms, but we can confirm
+that it boots on the following platforms:
+
+* ASUS KGPE-D16,
```

Comment:
@EduKav1813 @macpijan if I remember correctly, we should drop this from the list and DTS logic

Reply:
https://github.com/Dasharo/dts-scripts/pull/50 - PR removing ASUS support in DTS

Reply:
I am not sure, but here is the commit that removes this hardware from the list: db9a619e36317c3f4930693677637f379eb21105

We can revert it if not needed